lib/public/AppFramework/Http/TextPlainResponse.php

```diff
@@ -38,14 +42,14 @@ class TextPlainResponse extends Response {
 	/**
 	 * constructor of TextPlainResponse
 	 * @param string $text The text body
-	 * @param int $statusCode the Http status code, defaults to 200
+	 * @param S $statusCode the Http status code, defaults to 200
+	 * @param H $headers
 	 * @since 22.0.0
 	 */
-	public function __construct(string $text = '', int $statusCode = Http::STATUS_OK) {
-		parent::__construct();
+	public function __construct(string $text = '', int $statusCode = Http::STATUS_OK, $headers = []) {
```

Comment:
```suggestion
	public function __construct(string $text = '', int $statusCode = Http::STATUS_OK, array $headers = []) {
```

Can we type those new parameters or will that confuse Psalm again?

Reply:
Fixed

Reply:
(also for the $status)